x-pack/plugin/otel-data/src/yamlRestTest/resources/rest-api-spec/test/20_logs_tests.yml

```diff
@@ -70,3 +70,24 @@ setup:
   - match: { hits.hits.0.fields.error\.exception\.type: ["MyException"] }
   - match: { hits.hits.0.fields.error\.exception\.message: ["foo"] }
   - match: { hits.hits.0.fields.error\.stack_trace: ["Exception in thread \"main\" java.lang.RuntimeException: Test exception\n at com.example.GenerateTrace.methodB(GenerateTrace.java:13)\n at com.example.GenerateTrace.methodA(GenerateTrace.java:9)\n at com.example.GenerateTrace.main(GenerateTrace.java:5)"] }
+---
+"@timestamp should be used as sort field":
```

Comment:
I'm not sure about these tests. It feels like they may rely too much on an implementation details that _search returns documents in order of how they're indexed. I don't think there's a guarantee on the ordering of search results when no sorting or scoring is involved. We're not sorting because we want the search results to return in a specific order but because it has benefits wrt storage savings. A unit tests isn't the best way to verify that. So maybe just check that the index setting has been applied or even just remove the tests.

Reply:
Makes sense.

Overall I always prefer to at least have some test coverage, so I adapted the test which now checks if the settings are applied.